### DISCUSSION TITLE
The skin editor should also be able to configure touch input (for all gamemodes)

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#21011] [Ideas]
I've seen this requested partially for mania, but I think this should apply to all modes. Allow setting certain regions for specific controls, such as a specific region of the screen dedicated to keys for standard, specific (custom) regions for left, right, and dash for catch, custom region for taiko, and obviously customizability for mania. This would allow for on screen visual buttons that actually have function, and arent limited to the default regions.

**Comment by bdach** [comment]
Was the plan all along.
